cargo-dist/tests/snapshots/axolotlsay_basic.snap

```diff
@@ -1,7 +1,6 @@
 ---
 source: cargo-dist/tests/gallery/dist/snapshot.rs
 expression: self.payload
-snapshot_kind: text
```

Comment:
probably due to the minijinja update? idk

Reply:
What version of cargo-insta do you have? This was added when Ellen updated a test, which makes me think they might have a newer version than you or I do.

Reply:
OK yeah, this field landed in cargo-insta 1.41 https://github.com/mitsuhiko/insta/commit/0b81773e97535f1960c1443f85b491adceb36dd6

Prior to that insta only had a single snapshot format so there was no need to specify.

Reply:
I’ll upgrade locally!